docs/element-definitions.md

```diff
@@ -1038,40 +1071,6 @@ The following shows a link with the relation of `foo` and the URL of `/bar`.
 }
 ```
 
-## Profiles
-
-The primary means by which users can provide semantic definitions and other meta information is through a profile. A profile MAY provide semantic information about an element and its data, it MAY provide specific instructions about elements such as how inheritance should work or how elements should be processed, and it MAY be used to modify understanding of existing elements in other profiles. The usage of a profile is not limited to these uses here, and SHOULD be left up to the profile author to define its use.
-
-To point to a profile, you MAY use the [profile link relation](https://www.ietf.org/rfc/rfc6906.txt) as a meta link in your root element or in any other element. Profile links may also be found outside of the document itself in places like the [HTTP Link Header](http://www.w3.org/wiki/LinkHeader). Additionally, a profile link is not required in order to use the functionality a profile provides, as a media type MAY define the same things a profile.
-
-Below is an example of how a profile link is used as a meta link.
-
-```json
-{
-  "element": "foo",
-  "meta": {
-    "links": [
-      {
-        "element": "link",
-        "attributes": {
-          "relation": {
-            "element": "string",
-            "content": "profile"
-          },
-          "href": {
-            "element": "string",
-            "content": "http://example.com/profiles/foo"
-          }
-        }
-      }
-    ]
-  },
-  "content": "bar"
-}
-```
-
-The example shows a `foo` element with a `profile` link. This profile link informs the parser this particular element is defined as part of the linked profile.
-
```

Comment:
This block shouldn't be moved or changed.

Reply:
It doesn't belong to API Elements too. So, it shouldn't be place at the bottom after API Elements. @kylef What do you think?

Reply:
Yes I don't think it belongs here, the new placement is better as it is right next to the "Extending API Elements" section. Since "Extending API Elements" talks about profiles, perhaps it should go above it, or be incorporated in the extended API Elements section. So Extending API Elements shows profiles which can provide additional semantics on any element and then talks about the extension element.

![screen shot 2018-07-09 at 11 41 33](https://user-images.githubusercontent.com/44164/42446116-0fccbbf6-836d-11e8-9ef8-4a08f43e703a.png)